pegasus/sites.v3/code.org/views/donor_slider.haml

```diff
@@ -33,3 +26,7 @@
             = "#{supporter[:name_s]}"
           - else
             = "#{supporter[:name_s]}, "
+    
+    .donor-button
+      %button{onclick: "window.location.href = '/donate'"}
```

Comment:
Is there a reason why we do the navigation with JS rather than just using a link?  This seems less accessible...

Reply:
I referred to this doc in terms of my decision
https://www.deque.com/blog/accessible-aria-buttons/

Reply:
Also I've heard in the past it's best to be as syntactically true to what you are creating as possible. Since this visibly is a button, I wanted to use the respective HTML element accessibly. I'm open to other recommendations as well

Reply:
I defer to @epeach for this kind of issue, but from reading a few other pages about accessibility, it sounds like `<a>` should be used for going to a new page.  If we use JS, we are probably losing the ability of the browser to preview the link, the ability of search engines to follow it, and so on.  I think the page referred to compares `<button>` to `<a href=”#”>`, which is kind of a special case?

Reply:
Reference: https://ux.iu.edu/writings/buttons-vs-links-basic/#:~:text=There%20are%20differences%20as%20to,affect%20the%20website%20at%20all and https://accessibleweb.com/question-answer/is-it-ok-to-have-a-link-that-looks-like-a-button/#:~:text=Yes%2C%20it%20is%20OK%20to,link%20elements%20function%20as%20buttons.
Unfortunately (fortunately?), there is no right answer here. Since there is not consensus, my personal preference is to go with button (with appropriate tags to make sure it is read as a link by screenreaders - see the second reference link). I think current UI design has moved away from link-as-nav, button-as-action, so for accessibility, I'm inclined to lean towards using the semantic elements with appropriate accessibility attributes.

Reply:
From these two links, it does sound to me as though we should use an `<a>` tag for the link itself.